go.mod

```diff
@@ -61,9 +61,9 @@ require (
 	golang.org/x/term v0.15.0
 	golang.org/x/time v0.0.0-20220922220347-f3bd1da661af
 	gonum.org/v1/gonum v0.11.0
-	google.golang.org/genproto/googleapis/rpc v0.0.0-20230711160842-782d3b101e98
-	google.golang.org/grpc v1.58.3
-	google.golang.org/protobuf v1.31.0
+	google.golang.org/genproto/googleapis/rpc v0.0.0-20231002182017-d307bd883b97
+	google.golang.org/grpc v1.60.1
```

Comment:
Can we update this to `v1.62.0`? (or at least `v1.61.0`). I think we want to include this fix for a regression introduced in `v1.60.0`: https://github.com/grpc/grpc-go/pull/6922